release/s3_sync.sh

```diff
@@ -28,9 +28,19 @@ else
   PUBLIC_READ=''
 fi
 cd ${BASE_DIRECTORY}
-DEST_DIR=kubernetes-${RELEASE_BRANCH}/releases/${RELEASE}/artifacts
-if [ ! -z "${REPO}" ]
+BASE_DIR=kubernetes-${RELEASE_BRANCH}
```

Comment:
Can we name this something along the lines of PREFIX_DIR or RELEASE_BASE_DIR?

Reply:
Done